lib/uffizzi/cli/project.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'io/console'
+require 'uffizzi'
+require 'uffizzi/auth_helper'
+require 'uffizzi/response_helper'
+require 'thor'
+
+module Uffizzi
+  class CLI::Project < Thor
+    include ApiClient
+
+    desc 'compose', 'compose'
+    method_option :file, required: false, aliases: '-f'
+    require_relative 'project/compose'
+    subcommand 'compose', Uffizzi::CLI::Project::Compose
+
+    desc 'list', 'list'
+    def list
+      run('list')
+    end
+
+    private
+
+    def run(command)
+      return Uffizzi.ui.say('You are not logged in.') unless Uffizzi::AuthHelper.signed_in?
+
+      case command
+      when 'list'
+        handle_list_command
+      end
+    end
+
+    def handle_list_command
+      hostname = ConfigFile.read_option(:hostname)
+      response = fetch_projects(hostname)
+
+      if ResponseHelper.ok?(response)
+        handle_succeed_response(response)
+      else
+        handle_failed_response(response)
+      end
+    end
+
+    def handle_failed_response(response)
+      print_errors(response[:body][:errors])
+    end
```

Comment:
I think we can move it to the response helper, because it's going to be repeated in almost every command